shared/src/components/Product/productCopies.tsx

```diff
@@ -439,4 +439,103 @@ export const productCopies: { [vault in VaultOptions]: ProductCopies } = {
       ),
     },
   },
+  "ryvUSDC-ETH-P-THETA": {
+    title: "T-yUSDC-P-ETH",
+    subtitle: "ETH Put-Selling",
+    description:
+      "Generates yield by running an automated ETH put selling strategy.",
+    tags: ["PUT-SELLING"],
+    strategy: (
+      <>
+        This vault earns yield on its yvUSDC deposits by running an automated
```

Comment:
Let's add a link to yvUSDC vault page https://yearn.fi/invest/0x5f18C75AbDAe578b483E5F43f12a39cF75b973a9

Reply:
Updated with latest copies from Vinnie